dockers/docker-pde/docker_init.sh

```diff
@@ -10,6 +11,8 @@ start_bcm()
 
 PLATFORM_DIR=/usr/share/sonic/platform
 HWSKU_DIR=/usr/share/sonic/hwsku
+# Default use python2 version
+SONIC_PLATFORM_API_PYTHON_VERSION=2
```

Comment:
Why is this defaulting to Python 2? Also, where is `SONIC_PLATFORM_API_PYTHON_VERSION` used?

Reply:
This is just default value. If you see the following lines, the value will be assigned to new version.

Geans


# If the Python 3 sonic-platform package is not installed, try to install it
python3 -c "import sonic_platform" > /dev/null 2>&1 || pip3 show sonic-platform > /dev/null 2>&1
if [ $? -ne 0 ]; then
    SONIC_PLATFORM_WHEEL="/usr/share/sonic/platform/sonic_platform-1.0-py3-none-any.whl"
    echo "sonic-platform package not installed, attempting to install..."
    if [ -e ${SONIC_PLATFORM_WHEEL} ]; then
       pip3 install ${SONIC_PLATFORM_WHEEL}
       if [ $? -eq 0 ]; then
          echo "Successfully installed ${SONIC_PLATFORM_WHEEL}"
          SONIC_PLATFORM_API_PYTHON_VERSION=3
       else
          echo "Error: Failed to install ${SONIC_PLATFORM_WHEEL}"
       fi
    else
       echo "Error: Unable to locate ${SONIC_PLATFORM_WHEEL}"
    fi
else
    SONIC_PLATFORM_API_PYTHON_VERSION=3
fi

Reply:
All successful code paths are setting `SONIC_PLATFORM_API_PYTHON_VERSION` to 3 anyways, so I'd recommend just setting `SONIC_PLATFORM_API_PYTHON_VERSION=3` here.

Reply:
Done,  change default to 3 now